src/renderer/store/create.ts

```diff
@@ -15,8 +15,9 @@ export default (initialState = {}) => {
   const store = createStore(
     reducers,
     initialState,
-    composeWithDevTools(
-      applyMiddleware(...[errorsMiddleware, createDebounce(), sagaMiddleware, thunk, promise()])
+    compose(
+      applyMiddleware(...[errorsMiddleware, createDebounce(), sagaMiddleware, thunk, promise()]),
+      DevTools.instrument()
```

Comment:
we probably should only add this if `process.env.NODE_ENV === 'development'` ,right?